README.md

```diff
@@ -1,7 +1,7 @@
 # Node Maintenance Operator (NMO)
 
 <p align="center">
-<img width="100" src="config/manifests/nmo_blue_icon.png">
+<img width="100" src="config/assests/nmo_blue_icon.png">
```

Comment:
There is an additional `s` in asse**s**ts -> New PR, #72, will fix that to `assets`.